src/main/java/com/kodilla/ecommercee/repository/GroupDao.java

```diff
@@ -0,0 +1,23 @@
+package com.kodilla.ecommercee.repository;
+
+import com.kodilla.ecommercee.domain.Group;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import javax.transaction.Transactional;
+import java.util.List;
+import java.util.Optional;
+
+@Transactional
+@Repository
+public interface GroupDao extends CrudRepository<Group, Long> {
+
+    @Override
+    List<Group> findAll();
+
+    @Override
+    Optional<Group> findById(Long userId);
+
+    @Override
+    void deleteById(Long groupId);
```

Comment:
W GroupController nie bylo opcji Delete group - pytanie czy bedzie to wykorzystywne.